nevergrad/optimization/callbacks.py

```diff
@@ -135,3 +135,35 @@ def load_flattened(self, max_list_elements: int = 24) -> List[Dict[str, Any]]:
                         break
                     flat_data[-1][key + "#" + "_".join(str(i) for i in indices)] = value
         return flat_data
+
+    def to_hiplot_experiment(self, max_list_elements: int = 24) -> tp.Any:  # no typing here seems Hiplot is not a hard requirement
```

Comment:
```suggestion
    def to_hiplot_experiment(self, max_list_elements: int = 24) -> tp.Any:  # no typing here since Hiplot is not a hard requirement
```